components/UnderConstruction.tsx

```
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export function UnderConstruction() {
+  return (
+    <div className="flex min-h-screen flex-col px-4 py-12">
+      <div className="mb-8 flex flex-col items-center">
+        <svg width="80" height="80" fill="none" viewBox="0 0 80 80" className="mb-4">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)